app/views/admin/booths/_change_state_dropdown.html.haml

```diff
@@ -1,7 +1,9 @@
 - if booth.transition_possible? :accept
-  %li= link_to 'Accept booth',
-  accept_admin_conference_booth_path(@conference.short_title, booth),
-  method: :patch, id: "accept_booth_#{booth.id}"
+  - unless @conference.maximum_accepted_booths?
```

Comment:
This should be added in admin_ability.rb, and then here you can simply do `if can? :accept, booth`